docs/src/pages/[platform]/connected-components/geo/react.mdx

```diff
@@ -21,7 +21,7 @@ There's a known issue for users of Create React App v4 where the default prod `b
 
 > Prerequisites:
 >
-> 1. Install`@aws-amplify/ui-react-geo`with [npm](https://www.npmjs.com/) or [yarn](https://yarnpkg.com/). ( e.g. `npm install @aws-amplify/ui-react-geo`)
+> 1. Install `@aws-amplify/ui-react-geo` and `@aws-amplify/geo` with [npm](https://www.npmjs.com/) or [yarn](https://yarnpkg.com/). ( e.g. `npm install @aws-amplify/ui-react-geo @aws-amplify/geo`)
```

Comment:
IMO an explicit install command would align better with the rest of the docs, thoughts?

Reply:
Let's address in a follow up